deploy/olm-catalog/ocs-operator/0.0.1/ocs-operator.v0.0.1.clusterserviceversion.yaml

```diff
@@ -274,7 +194,8 @@ spec:
                   valueFrom:
                     fieldRef:
                       fieldPath: metadata.namespace
-                image: rook/ceph:v1.0.1
+                image: rook/ceph:master
```

Comment:
I'd suggest we use the most recent master [tag](https://hub.docker.com/r/rook/ceph/tags/) (right now it is `rook/ceph:v1.0.0-422.g27cad9d`) instead of the `master` tag. This way we can control what we're testing. Then we should regularly update it (every few days?) until we branch/tag for v1.1. Rook doesn't add the `imagePullPolicy: Always` to the pods it starts, so a cluster could end up in an unpredictable state with the `master` tag.

Reply:
that crossed in mid-air. we can take that up separately, @travisn